src/components/TalentsListPage/Filter/Filter.module.scss

```
@@ -0,0 +1,32 @@
+.filters {
+    display: flex;
+    flex-direction: column;
+    margin-top: 50px;
+    width: 310px;
+}
+.caption {
```

remove this rule from this file and rename class and selector like this
```suggestion
.page_title {
```